### PR TITLE
Updated readme file to describe current condition more accurately.

### PR DESCRIPTION
**Pull request by adityaruplaha**
Most games now run to a "playable" state. Updated License version to 3.

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/3008)

### CODE REVIEW

**Comment by wwylele** [comment]
License version is not something one could freely "update". Please revert it.

**Comment by adityaruplaha** [comment]
Ok, reverted.

File: README.md [hunk 1]
```diff
@@ -5,13 +5,15 @@ Citra Emulator
...
 
-Citra is an experimental open-source Nintendo 3DS emulator/debugger written in C++. It is written with portability in mind, with builds actively maintained for Windows, Linux and macOS. Citra only emulates a subset of 3DS hardware and therefore is generally only useful for running/debugging homebrew applications. At this time, Citra is even able to boot several commercial games! Most of these do not run to a playable state, but we are working every day to advance the project forward.
+Citra is an experimental open-source Nintendo 3DS emulator/debugger written in C++. It is written with portability in mind, with builds actively maintained for Windows, Linux and macOS. Citra only emulates a subset of 3DS hardware and therefore is generally only useful for running/debugging homebrew applications.
+
+Citra is even able to boot several commercial games! Some of of these do not run to a playable state, but most do, and we are working every day to advance the project forward. <b>Playable here means the same as "OK" in our [game compatibility list](https://citra-emu.org/game).</b>
```
Comment (Dragios) [inline README.md]: able to boot several commercial games?
That statement is definitely inaccurate now.

**Comment by Dragios** [comment]
Also remember to fix clang formatting.
You may ask for guidance in discord for this matter.

File: README.md (hunk 1 above)
Comment (baka0815) [inline README.md]: ``Some of of these do`` there is one "of" too much.

Comment (adityaruplaha) [reply README.md]: ok, I'll fix that.

Comment (adityaruplaha) [reply README.md]: Well, i did not change that.

**Comment by adityaruplaha** [comment]
fixed typos in my last commit.

File: README.md (hunk 1 above)
Comment (Dragios) [reply README.md]: I know, so change it now to better reflect the current situation.

**Pull request merged**